documentation/components/tutorial-flow-grid.asciidoc

```diff
@@ -985,27 +1004,21 @@ Styling of the grid itself can be done within custom styles by overriding grid-s
 </dom-module>
 ----
 
-The _theme-for="vaadin-grid"_ indicates that it's overriding vaadin-grid -components styling. In the the css the
-_:host(.styled)_ is a selector for vaadin-grid that has "styled" as class. So outside of shadow dom that would be
-_vaadin-grid.styled_, but the shadow dom is boxed in it's own dom (ergo, the shadow dom) it must be selected with
-:host([selector]).
-
-In this example we're setting special styles for vaadin-grid with "styled" class. A grid without "styled" class remains
-normal.
+* This sets custom styles for a `vaadin-grid` with a "styled" class. Grid's without this class remain normal.
+* `theme-for="vaadin-grid"` indicates that it is overriding `vaadin-grid -components` styling. 
+* `:host(.styled)` is a selector for `vaadin-grid` that has "styled" as a class. Outside the shadow DOM this is `vaadin-grid.styled`, but because the shadow DOM is boxed in its own DOM, it is selected with `:host([selector])`.
 
-=== Styling with variables
 
-Although the shadow-dom is boxed and cannot be affected outside, there's one method to pass info to the shadow-dow:
-CSS variables.
+=== Styling with CSS Variables
```

Comment:
I find it quite weird that this is in Grid documentation, because this has nothing to do with Grid. But let's leave it here now and I'll create an issue to relocate it to generic styling documentation, if that even exists 🤔

Reply:
Noted